.github/workflows/OCV-Contrib-PR-4.x-W10.yaml

```diff
@@ -20,7 +20,7 @@ env:
   TARGET_BRANCH_NAME: ${{ github.base_ref }}
   OPENCV_TEST_DATA_PATH: ${{ github.workspace }}\opencv_extra\testdata
   OPENCV_TEST_REQUIRE_DATA: 1
-  GTEST_FILTER_STRING: '-Samples.findFile:videoio/videocapture_acceleration.read/122:videoio/videocapture_acceleration.read/126:videoio/videocapture_acceleration.read/136'
+  GTEST_FILTER_STRING: '-Samples.findFile:videoio/videocapture_acceleration.read/yuv420p_libvpxxvp9_mp4_MSMF_NONE_MAT'
```

Comment:
There are two cases in the original suppression list and only one in thew new here and bellow.

Reply:
Yes, but since test builds has passed twice with updated filter, so I decided that we can use it.

https://github.com/opencv/opencv/actions/runs/13438427883/job/37699806975